docs/eth_typing.rst

```diff
@@ -49,6 +49,30 @@ Class that contains the different names used to represent hard forks on the Ethe
         Constantinople = 'Constantinople'
         Metropolis = 'Metropolis'
 
+
+ChainId
```

Comment:
```suggestion
ChainID
```

Or, conversely, I'd support renaming the `IntEnum` class `ChainId`

Reply:
Renamed the enum, feels closer to convention.